desktopexporter/internal/server/server.go

```diff
@@ -122,8 +119,8 @@ func (s *Server) traceIDHandler(writer http.ResponseWriter, request *http.Reques
 	}
 }
 
-func indexHandler(writer http.ResponseWriter, request *http.Request) {
-	if os.Getenv("SERVE_FROM_FS") == "true" {
+func (s *Server) indexHandler(writer http.ResponseWriter, request *http.Request) {
```

Comment:
I see this little fix. Yay consistency!

Reply:
Busted! The fix was actually to access a server variable, but yeah it works for consistency too.